src/java/org/jivesoftware/openfire/labelling/Clearance.java

```diff
@@ -0,0 +1,14 @@
+package org.jivesoftware.openfire.labelling;
+
+import org.dom4j.Element;
+
+/**
+ * Created by dwd on 16/03/17.
+ */
+public class Clearance {
+    Element element;
+
+    Clearance(Element element) {
+        this.element = element;
+    }
+}
```

Comment:
I think this isn't even used, actually. I originally intended making a wrapper construct much like the SecurityLabel, to contain clearances within a syntax-marked XML wrapper. I'll ditch this.